Run args (#438)
garnix-io/garn: website/src/docs/concepts.mdx

You can run `Executable`s with `garn run`.

If you would like `garn run` to pass arguments to your executable, use `"$@"`

> where you'd like to receive them. For examples

Reviewer: ```suggestion
where you'd like to receive them. For example:
```

```typescript
import * as garn from "https://garn.io/ts/v0.0.16/mod.ts";

export const hello =

>   garn.emptyEnvironment.shell('printf "My first two arguments were %s and %s" "$@"');

Reviewer: ```suggestion
  garn.shell('printf "My first two arguments were %s and %s" "$@"');
```

That should work, right?

```

> This works exactly like bash-script arguments, so it helps to know the
> differences between `"$@"`, `$@`, and `$*`.

Reviewer: This feels a bit ... condescending? So maybe
```suggestion
This works exactly like bash-script arguments. For the exact behavior, see `man bash`, section `PARAMETERS`.
```
